app/views/docs/command-line-ci.phtml

```diff
@@ -0,0 +1,20 @@
+<p>
+The Appwrite CLI can be used in a non-interactive and headless manner, without saving configuration or sessions. This is especially useful when you want to automate tasks on a CI server.
+</p>
+<h2><a href="/docs/commandline-ci#CIMode" id="CIMode">CI Mode</a></h2>
+
+<p>You can enable CI mode for the Appwrite CLI by setting the <b>project ID, endpoint, and API Key</b>:</p>
+
+<div class="ide margin-bottom" data-lang="bash" data-lang-label="CLI">
+    <pre class="line-numbers"><code class="prism language-bash" data-prism>appwrite client \
+--endpoint https://[HOSTNAME_OR_IP]/v1 \
+--projectId [YOUR_PROJECT_ID] \
+--key YOUR_API_KEY</code></pre>
+</div>
+
+<p>When you set the global configuration parameters using the <code>appwrite client</code> command, they take precedence over the local configuration parameters in your <code>appwrite.json</code> thereby switching the CLI to non-interactive mode. </p> 
+
+<p>In this mode, the CLI can only interact with one project at a time.</p>
+
```

Comment:
Is it worth it to add some tips on how to deploy a function non-interactively? 

For functions, you can pass the function ID and a flag to say yes to all warning:

```
$ appwrite deploy function -h
Usage: appwrite deploy function [options]

Deploy functions in the current directory.

Options:
  --functionId <functionId>  Function ID
  --all                      Flag to deploy all functions
  --yes                      Flag to confirm all warnings
  -h, --help                 display help for command
```

The `--yes` flag is also available for `appwrite deploy collection`:

```
$ appwrite deploy collection -h
Usage: appwrite deploy collection [options]

Deploy collections in the current project.

Options:
  --all       Flag to deploy all functions
  --yes       Flag to confirm all warnings
  -h, --help  display help for command
```

(looks like the description on the `--all` flag for `appwrite deploy collection` needs to be fixed 😅)

Reply:
Okay :)